features-json/background-sync.json

```diff
@@ -0,0 +1,330 @@
+{
+  "title":"Background Sync API",
+  "description":"Provides one-off and periodic synchronization for Service Workers with an onsync event.",
+  "spec":"https://wicg.github.io/BackgroundSync/spec/",
+  "status":"wd",
+  "links":[
+    {
+      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=1217544",
+      "title":"Mozilla bug report"
+    },
+    {
+      "url":"https://developer.mozilla.org/docs/Web/API/SyncManager",
+      "title":"Documentation"
+    },
+    {
+      "url":"https://developer.microsoft.com/en-us/microsoft-edge/platform/status/backgroundsyncapi/",
+      "title":"Edge status"
+    },
+    {
+      "url":"https://wicg.github.io/BackgroundSync/spec/",
+      "title":"Working draft or equivalent"
+    }
+  ],
+  "bugs":[
+    
+  ],
+  "categories":[
+    "JS",
```

Comment:
Invalid trailing comma here. You can use http://jsonlint.com/ to validate.

Reply:
done... Time to update the error messages ;)